GFS_layer/GFS_typedefs.F90

```diff
@@ -2401,6 +2404,9 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
     logical              :: do_z0_hwrf17   = .false.                  !< flag for using z0 scheme in 2017 HWRF
     logical              :: do_z0_hwrf17_hwonly = .false.             !< flag for using z0 scheme in 2017 HWRF only under high wind
     real(kind=kind_phys) :: wind_th_hwrf   = 33.                      !< wind speed threshold when z0 level off as in HWRF
+    real(kind=kind_phys) :: alpha_stable   = 5.                       !< TBD
+    real(kind=kind_phys) :: alpha_unstable = 16.                      !< TBD
```

Comment:
Can you add these descriptions, and perhaps note these were added by Sofar and the date.